examples/paged_query.rs

```diff
@@ -68,10 +89,72 @@ fn fill_table(session: &CurrentSession) {
     }
 }
 
+fn paged_with_value(session: &CurrentSession) {
+    let create_table_cql =
+        "CREATE TABLE IF NOT EXISTS test_ks.another_test_table (a int, b int, c int, d int, e int, primary key((a, b), c, d));";
+    session
+        .query(create_table_cql)
+        .expect("Table creation error");
+
+    for v in 1..=10 {
+        session
+            .query_with_values("INSERT INTO test_ks.another_test_table (a, b, c, d, e) VALUES (?, ?, ?, ?, ?)",
+                               AnotherTestTable {
+                                   a: 1,
+                                   b: 1,
+                                   c: 2,
+                                   d: v,
+                                   e: v,
+                               }.into_query_values(),
+            ).unwrap();
+    }
+
+
+    let q = "SELECT * FROM test_ks.another_test_table where a = ? and b = 1 and c = ?";
+    let mut pager = session.paged(3);
+    let mut query_pager = pager.query(q, Some(query_values!(1, 2)));
+
+    // Oddly enough, this returns false the first time...
+    assert!(!query_pager.has_more());
+
+    let mut assert_amount = |a| {
+        let rows = query_pager.next().expect("pager next");
+
+        assert_eq!(a, rows.len());
+    };
+
+    assert_amount(3);
+    assert_amount(3);
+    assert_amount(3);
+    assert_amount(1);
+
+    assert!(!query_pager.has_more());
+}
+
+// TODO: Why does this throw 'Expected 4 or 0 byte int (52)'
+fn paged_with_values_list(session: &CurrentSession) {
+    let q = "SELECT * FROM test_ks.my_test_table where key in (?)";
+    let mut pager = session.paged(2);
+    let mut query_pager = pager.query(q, Some(query_values!(vec![100, 101, 102, 103, 104, 105])));
```

Comment:
please try query values without `vec!`, just like `query_values!(100, 101, ...)`
I think it may solve the error you mentioned in a TODO comment